resources/kernel/make_kernel.sh

```diff
@@ -149,20 +149,21 @@ kernel_binary() {
 #   make_kernel
 #       /path/to/kernel/dir \
 #       format              \
+#       [target]            \
 #       [num_cpus_build]    \
 #       [/path/to/kernel/destination]
 make_kernel() {
     kernel_dir="$1"
     format="$2"
-    nprocs="$3"
-    dst="$4"
+    target="$3"
```

Comment:
Hmm, why is this added as an extra param versus how things were before?

Reply:
Because this function can be recycled to build `.deb`s with `make deb-pkg`. So the `target` is no longer a variable of the kernel format.